spec/requests/schedule_proposal_spec.rb

```diff
@@ -72,6 +74,10 @@
   end
 
   describe "DELETE /documents/:document/schedule-proposal" do
+    before { stub_any_publishing_api_put_content }
+
+    let(:edition) { create(:edition) }
```

Comment:
Can you help me understand where this has come from?

Reply:
Argh sorry I think that's just a copy and paste error.

Reply:
Sorted